ambry-store/src/main/java/com.github.ambry.store/Log.java

```diff
@@ -34,13 +39,15 @@
 class Log implements Write {
   private final String dataDir;
   private final long capacityInBytes;
+  private final boolean isLogSegmented;
   private final StoreMetrics metrics;
+  private final Iterator<Pair<String, String>> segmentNameAndFileNameIterator;
```

Comment:
Can you add docs to both these data structures as to what purpose they serve and how they are expected to evolve ?